src/Nethermind/Nethermind.Init/Steps/RegisterRpcModules.cs

```diff
@@ -27,7 +27,7 @@
 
 namespace Nethermind.Init.Steps;
 
-[RunnerStepDependencies(typeof(InitializeNetwork), typeof(SetupKeyStore), typeof(InitializeBlockchain), typeof(InitializePlugins))]
+[RunnerStepDependencies(typeof(InitializeNetwork), typeof(SetupKeyStore), typeof(InitializeBlockchain), typeof(InitializeBlockProducer), typeof(InitializePlugins))]
```

Comment:
why additional dependencies?

Reply:
`IHealthRpcModule` previously was created at `RegisterPluginRpcModules` which has access to block producer. Now it is in `RegisterRpcModules` so need to add block producer also.

Reply:
Not sure if it won't cause even more problems when trying to make RPC start as early as possible.